release.yaml

```diff
@@ -16,7 +16,7 @@ framework:
     codecoverage: true
     
   - artifact: dev.galasa.ras.couchdb
-    version: 0.25.0
+    version: 0.26.0
```

Comment:
28?

Reply:
Addressed in https://github.com/galasa-dev/extensions/pull/200/commits/0124979801009e10b7c4fc6f1bce4ef8583e247b